src/paper/tasks.py

```diff
@@ -1308,107 +1307,38 @@ def pull_new_openalex_works(page=0, retry=0):
     except Exception as e:
         sentry.log_error(e)
 
+    total_papers_processed = 0
+    cursor = "*" # used for pagination
     try:
         open_alex = OpenAlex()
-        works = open_alex.get_new_works(since_date=fetch_since_date)
-        if not works or len(works) == 0:
-            PaperFetchLog.objects.create(
-                source=PaperFetchLog.OPENALEX,
-                fetch_type=PaperFetchLog.FETCH_NEW,
-                status=PaperFetchLog.SUCCESS,
-                started_date=start_date,
-                completed_date=datetime.now(),
+        
+        while True:
```

Comment:
Is there a chance of this not terminating?

Reply:
I think it's unlikely given I'm checking `next_cursor` and for null or empty works response.